R/navs-legacy.R

```diff
@@ -171,12 +178,12 @@ navs_bar_ <- function(..., title = NULL, id = NULL, selected = NULL,
     theme = theme
   )
 
-  if (!is.null(bg)) {
-    # navbarPage_() returns a tagList() of the nav and content
-    navbar[[1]] <- tagAppendAttributes(
-      navbar[[1]], style = css(background_color = paste(bg, "!important"))
-    )
-  }
+  # navbarPage_() returns a tagList() of the nav and content
+  navbar[[1]] <- tagAppendAttributes(
+    navbar[[1]], 
+    style = if (!is.null(bg)) css(background_color = paste(bg, "!important")),
+    "data-bs-theme" = navbar_color_mode
```

Comment:
This way we could get rid of `navbar_color_mode`?

```suggestion
    "data-bs-theme" = if (inverse) "dark" else "light"
```

Reply:
Part of the problem I'm trying to solve is that this isn't really a binary choice, at minimum we want to support three different use-cases:

1. A light background navbar with `data-bs-theme="light"`
2. A dark background navbar with `data-bs-theme="dark"`
3. A navbar that follows the global `data-bs-theme` setting, which we achieve by setting `data-bs-theme="auto"`.

We won't depend on `"auto"` directly; we'll rely on the fact that it's not a known color mode, in which case the global color mode can affect the navbar colors.